assets/course-theme/themes/default-theme.scss

```diff
@@ -0,0 +1,116 @@
+
+:root {
+	--header-progress-bar-height: 4px;
+	--sidebar-width: 300px;
+	--content-padding: 32px;
+}
+
+$breakpoint: 783px;
+
+// Fixed sidebar.
+
+.sensei-course-theme {
+	&__sidebar {
+		position: fixed;
+		top: 0;
+		bottom: 0;
+		left: 0;
+		width: var(--sidebar-width);
+		border-right: 1px solid var(--border-color);
+		background-color: var(--bg-color);
+		overflow: auto;
+		overscroll-behavior: contain;
+		display: flex;
+		flex-direction: column;
+		scrollbar-gutter: stable both-edges;
+		padding: 12px;
+
+	}
+
+	&__header ~ &__columns &__sidebar {
+		top: var(--full-header-height) !important;
+	}
+
+	&__main-content ~ &__sidebar {
+		left: unset;
+		right: 0;
+		border-right: unset;
+		border-left: 1px solid var(--border-color);
+
+	}
+
+	&__main-content {
+		padding: 32px 0;
+		margin-right: var(--sidebar-width) !important;
+
+		// A Divi fix since it doesn't load Gutenberg block library styles.
+		@at-root {
+			.et_pb_pagebuilder_layout & {
+				flex: 1;
+			}
+		}
+	}
+
+	&__sidebar ~ &__main-content {
+		margin-right: unset ! important;
+		margin-left: var(--sidebar-width) !important;
+	}
+
+}
+
+// Mobile.
+
+@media screen and (max-width: ($breakpoint - 1)) {
```

Comment:
`min-width` and `max-width` are inclusive, not `less/greater than` but `less/greater than or equal`. So the less-than needs to be a pixel smaller, otherwise both rules would match.

And the variable used for both:
`min-width: $breakpoint` → screens above 783px, including exactly 783px.
`max-width: ($breakpoint - 1)` → screens below 782px, including exactly 782px.

Reply:
Ahh, got it. Thanks for clarifying. 👍